.github/workflows/dotnet-core-build-and-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
why ubuntu?

our app test can run on windows machines? in these machines windows dotnet is already installed, so you don't need to install separately